pkgs/stdenv/generic/default.nix

```diff
@@ -54,7 +61,11 @@ let
       pos' = if pos != null then "‘" + pos.file + ":" + toString pos.line + "’" else "«unknown-file»";
 
       throwEvalHelp = unfreeOrBroken: whatIsWrong:
-        assert (unfreeOrBroken == "Unfree" || unfreeOrBroken == "Broken");
+        assert (builtins.elem unfreeOrBroken [ "Unfree"
+                                             , "Broken"
```

Comment:
no comma in lists :grin:. If you use sublime text or Atom you can use my Nix highlighter to help you.